### PR TITLE
Dev issue 35 label left

### PR DESCRIPTION
**Pull request by zeroasterisk**
The `modifyProps` function is just an idea, but easiest to communicate this way....

### CODE REVIEW

**Comment by radekmie** [comment]
I'm generally against such _wrappers_, so how about this:

```js
const Bool = ({label, labelBefore, ...props}) =>
    <FormGroup label={labelBefore} {...props}>
        <section className={classnames(props.inputClassName, `checkbox${props.inline ? '-inline' : ''}`)}>
            <label htmlFor={props.id}>
                <input
                    checked={props.value}
                    disabled={props.disabled}
                    id={props.id}
                    name={props.name}
                    onChange={() => props.onChange(!props.value)}
                    type="checkbox"
                />&nbsp;
                {label}
            </label>
        </section>
    </FormGroup>
;
```

It will pass your `labelBefore` (or as you already named it `labelLeft`) to the `<FormGroup />`, and leave it's current label,

File: packages/uniforms-bootstrap3/src/components/fields/BoolField.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Bool = props =>
                     name={props.name}
                     onChange={() => props.onChange(!props.value)}
                     type="checkbox"
-                />
+                />&nbsp;
                 {props.label}
```
Comment (radekmie) [inline packages/uniforms-bootstrap3/src/components/fields/BoolField.js]: Now `props.label` is always `undefined` - change it to `label`.

File: packages/uniforms-bootstrap4/src/components/fields/BoolField.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const Bool = props =>
                     name={props.name}
                     onChange={() => props.onChange(!props.value)}
                     type="checkbox"
-                />
+                />&nbsp;
                 {props.label}
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/components/fields/BoolField.js]: Same as above.

**Comment by zeroasterisk** [comment]
updated... better?

**Comment by radekmie** [comment]
Great!

**Pull request merged**